utils/generate_confusables.py

```diff
@@ -79,19 +82,30 @@ def main(args=sys.argv):
 //
 // This source file is part of the Swift.org open source project
 //
-// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Copyright (c) 2020 Apple Inc. and the Swift project authors
 // Licensed under Apache License v2.0 with Runtime Library Exception
 //
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
+//
+////////////////////////////////////////////////////////////////////////////////
+// WARNING: This file is manually generated from
+// utils/UnicodeData/confusables.txt and should not be directly modified.
+// Run utils/generate_confusables.py to regenerate this file.
+////////////////////////////////////////////////////////////////////////////////
 '''
         f.write(header)
         f.write("//===----------------------------------------------------")
         f.write("------------------===//\n\n")
```

Comment:
I wonder why this line (and the above) is not included in `header` 🤔

Reply:
I have moved it inside `header` now.